### ISSUE TITLE
java.io.IOException: Unable to open <path>

### ISSUE THREAD

**Issue by emilyttran** [stanfordnlp/CoreNLP#721]
I am trying to lemmatize text but I keep running into this issue even though I am not using pos-tagging:

`Caused by: `java.io.IOException: Unable to open` "edu/stanford/nlp/models/pos-tagger/english-left3words/english-left3words-distsim.tagger" as class path, filename or URL`

I checked to see if I downloaded the model jar, which I did. I also downloaded the most recent package as well. Also, for some reason Intellij isn't importing the model jar. IntelliJ recognizes all the other jars except for the model jar. I am not sure how to fix this

**Comment by J38** [comment]
The lemmatizer requires part of speech tags.

How did you add the models jar in IntelliJ.  It seems like that is the issue.

It's been a while since I did this :) 

...but I think that what I did is:

1.) go to Project Structure
2.) Click on Modules
3.) Add a directory with all of the models jars in it
     (Note I set this up as a Jar Directory when I was asked)

I've been able to run Stanford CoreNLP stuff within IntelliJ.
